brave/brave-ios
# Fix #7168: Add Brave News Widget for topics clustering

## Summary of Changes

This pull request fixes #7168 

## Submitter Checklist:


## Test Plan:

- Add Top News widgets on the home screen:
  - Small/single article: 15m rotation per article, refreshes after 60m
  - Medium/up to 2 articles per rotation: 15m rotation per set, refreshes after 60m
  - Large/up to 5 articles per rotation: 20m rotation per set, refreshes after 60m
- Add Top News widgets on the lock screen:
  - Rectangular/single article: 15m rotation per article, refreshes after 60m

Verify: 
- Rotations are correct, new articles appear
- Images on articles that have them are displayed
- News articles load in Brave when tapping on them

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
